docs/community/community.md

```diff
@@ -1,6 +1,41 @@
 # KubeRay Community Governance
 
-## Becoming a Committer
+There are 4 roles in the KubeRay community:
+
+* **Contributor**: A contributor is a member who contributes to the KubeRay project through code, documentation,
+PR reviews, answering questions, etc.
+* **Triager**: A triager has the permission to triage issues and PRs.
+* **Approver**: An approver can approve PRs but cannot merge them.
+* **Committer**: A committer can merge PRs and is responsible for the KubeRay project.
+
+## Mechanism
+
+### Becoming a Contributor
+
+To become a KubeRay contributor, you can contribute to KubeRay through code, documentation, PR reviews,
+answering questions, and more.
+It's helpful to join the KubeRay channel `#kuberay-questions` on the [Ray Slack workspace](https://github.com/ray-project/ray?tab=readme-ov-file#getting-involved)
+to get started. You can also read [ray-project/kuberay#1059](https://github.com/ray-project/kuberay/issues/1059)
+for tips on getting involved in the KubeRay community.
+
+In addition, I suggest adding the
```

Comment:
nit: don't use "I" in README 

 In addition, you can add the Ray / KubeRay OSS community Google meeting to your calendar to join the bi-weekly KubeRay community meetings.